### PR TITLE
fix: typos in `README`

### PR DESCRIPTION
**Pull request by leovct**
Some minor spelling mistakes :)

### CODE REVIEW

**Comment by mustermeiszer** [comment]
@cdamian is removing the rpc sectio right?

File: README.md [hunk 1]
```diff
@@ -39,12 +39,5 @@ the tests within a tests-dedicated Docker container or without a container.
 
 Visit https://polkadot.js.org/apps for inspection
 
-**Note**: To use a custom Substrate endpoint, first set the enviromment variable before running the tests:
+**Note**: To use a custom Substrate endpoint, first set the environment variable before running the tests:
 `export RPC_URL="http://example.com:9934"`
-
-### Adding support for new RPC methods
```
Comment (cdamian) [inline README.md:45]: Please re-add this.

**Review by cdamian — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (leovct) [reply README.md:45]: Oh sorry, didn't notice!

**Comment by cdamian** [comment]
Thank you for the contribution @leovct!

**Pull request merged**